source/common/config/well_known_names.cc

```diff
@@ -102,8 +106,22 @@ TagNameValues::TagNameValues() {
   // http.[<stat_prefix>.]fault.(<downstream_cluster>.)*
   addTokenized(FAULT_DOWNSTREAM_CLUSTER, "http.*.fault.$.**");
 
-  // listener.[<address>.]ssl.cipher.(<cipher>)
-  addRe2(SSL_CIPHER, R"(^listener\..*?\.ssl\.cipher(\.(<CIPHER>))$)");
+  // listener.[<address>.]ssl.ciphers.(<cipher>)
+  addRe2(SSL_CIPHER, R"(^listener\..*?\.ssl\.ciphers(\.(<CIPHER>))$)", ".ssl.ciphers.");
```

Comment:
could we used two addTokenized statements rather than the re2? I'd have to dig out the speed test but I think that would be faster and arguably more readable.

Reply:
If we use two rules, we make it so that the user can't configure some of them: https://github.com/envoyproxy/envoy/blob/bb8aa93c53c808065ba4207224d1a301d9713439/api/envoy/config/metrics/v3/stats.proto#L223